.gitignore

```diff
@@ -34,3 +34,4 @@ src/generated/
 /*.d.ts.map
 /*.js
 /*.js.map
+!/*.config.js
```

Comment:
did you mean `.config.cjs`? We could probably get away without this line, since the `.cjs` was not ignored.

Reply:
No, it is about `lint-staged.config.js` which uses Node ESM. I found that our `clean` command removes that file which is not good.